lib/features/composer/domain/repository/contact_repository.dart

```diff
@@ -0,0 +1,7 @@
+
+import 'package:model/model.dart';
+import 'package:tmail_ui_user/features/composer/domain/model/auto_complete_pattern.dart';
+
+abstract class ContactRepository {
+  Future<List<Contact>> getContactSuggestions(AutoCompletePattern autoCompletePattern);
```

Comment:
why not use `EmailAddress` to easy to merge with `getAutoComplete`

Reply:
I want it to be a common abstract for Contact so we can extend other contacts later. So I converted it to EmailAddress in interactor